common/navbar.js

```diff
@@ -59,7 +72,35 @@
             var parentNewTab = obj.newTab;
             // template the url
             // TODO: This is done here to prevent writing a recursive function (again) in `setConfigJSON()`
-            if (obj.url && isCatalogDefined(catalogId)) obj.url = ERMrest.renderHandlebarsTemplate(obj.url, null, {id: catalogId});
+            if (obj.url && isCatalogDefined(catalogId)) {
+                obj.url = ERMrest.renderHandlebarsTemplate(obj.url, null, {id: catalogId});
+
+                // same origin urls may be relative
+                function isChaiseApp (url) {
+                    var appNames = ["record", "recordset", "recordedit", "search", "login"];
+                    var path = "chaise/";
+                    if (chaiseConfig && typeof chaiseConfig.chaiseBasePath === "string") {
+                        var path = chaiseConfig.chaiseBasePath;
+                        // append "/" if not present
+                        if (path[path.length-1] !== "/") path += "/";
+                    }
+
+                    var isChaise = false;
+                    for (var i=0; i<appNames.length; i++) {
+                        var name = appNames[i];
+                        // path/appName exists in our url
+                        if (url.indexOf(path + name) !== -1) isChaise = true;
```

Comment:
Is it possible to be more specific for this check? Do we want to add this to any chaise urls? or chaise urls that are in the same origin? 
Assuming we're in `gudmap.org`, should we add these parameters to all of the following? 

- `gudmap.org/chaise/recordset/#1/schema:table`
- `dev.gudmap.org/chaise/recordset/#1/schema:table`
- `rebuildingakidney.org/chaise/recordset/#1/schema:table`
- `gudmap.org/~jchudy/chaise/recordset/#1/schema:table`